RecoLocalFastTime/FTLRecProducers/python/mtdRecHits_cfi.py

```diff
@@ -9,8 +9,8 @@
 
 _endcapAlgo = cms.PSet(
     algoName = cms.string("MTDRecHitAlgo"),
-    thresholdToKeep = cms.double(0.0425),    # MeV
-    calibrationConstant = cms.double(0.085), # MeV/MIP
+    thresholdToKeep = cms.double(0.0425),       # MeV
+    calibrationConstant = cms.double(0.023156), # MeV/fC
```

Comment:
I'm concerned this change could interfere with processing 11_0_X DIGI files for the TDR re-reco. Since those digis were simulated with the old calibration, using the new calibration in RECO could produce incorrect results. It might be better to hold this PR until 11_2_X.

Reply:
good point (and I forgot about the 11_0_X compatibility point).
I agree with the hold.
However, IIRC in the last ORP the proposal was to extend this compatibility range to 11_2_X as well.
Should we put a proc modifier here and in digi?

Reply:
It should interfere IMO, if only a reRECO is done. This could would allow a study of luminosity effects, but only if DIGI are reproduced again, which is not in the plan. A proc modifier might allow us to use this on newer versions of the detector for studies without waiting winter. I am not sure anyway how much more complex the could would become, @casarsa what do you think?

Reply:
I would roll back the conversion of energy to MIP units. The focus of this PR is on the time resolution. The conversion to fC is just an intermediate step, not essential at this stage. I was planning in advance and setting up things for a future and longer-term development when a pulse shape of the electronic signal is available.